README.md

```diff
@@ -1,19 +1,30 @@
+
 # Flowsynth #
 
-Flowsynth is a tool for rapidly modelling network traffic. Flowsynth can be used to generate text-based hexdumps of packets as well as native libpcap format packet captures.
+Flowsynth is a tool for rapidly modeling network traffic. Flowsynth can be used
```

Comment:
Broke long lines and fixed typo s/modelling/modeling/.

Reply:
We have been here before with the alleged "typo" ... See https://github.com/secureworks/flowsynth/pull/5.   Personally, I prefer the American style.

Reply:
Haha, I can live with reverting that one.

Reply:
Then there is this find:
https://github.com/secureworks/flowsynth/blob/49ba2c172812b110eb8af2ab778e4baecfedc15b/src/flowsynth.py#L3